install-manifests/docker-compose-traefix-tls-gzip/README.md

```diff
@@ -0,0 +1,29 @@
+# Hasura GraphQL Engine on Docker with Traefix (Reverse Proxy and Load Balancer)
+
+This Docker Compose setup runs [Hasura GraphQL Engine](https://github.com/hasura/graphql-engine) along with Postgres, Platypus (authentication), and Traefix (a reverse proxy and load balancer, it's officially classified as an edge router) using `docker-compose`.
```

Comment:
We can link to Platypus and Traefix documentations.

Reply:
Sure, this has been applied.